blog/content/post/2021-07-07-tidy-transcriptomics-manifesto/index.en.md

```diff
@@ -36,19 +36,19 @@ sequenceDiagrams:
 
 <img src="images/logo.png" alt="tidytranscriptomics_logo" width="100px" height="100px"/>
 
-In this first post, we want to introduce the concept of tidy transcriptomics, and the principles is moved by.
+In this first post, we want to introduce the concept of tidy transcriptomics and its principles is moved by.
```

Comment:
It should be either "we want to introduce the concept of tidy transcriptomics and its principles"
or "we want to introduce the concept of tidy transcriptomics and the principles it's moved by"
(the first sounds better to me)